ci-operator/config/project-codeflare/codeflare-operator/project-codeflare-codeflare-operator-main.yaml

```diff
@@ -28,6 +28,72 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: codeflare-operator-e2e
+      commands: |
+          podman run -d -p 5000:5000 --name registry registry:2.8.1
+          export REGISTRY_ADDRESS=$(hostname -i):5000
+
+          KUBERAY_VERSION=$(make get-kuberay-version)
+          echo Deploying KubeRay ${KUBERAY_VERSION}
+          kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/default?ref=${KUBERAY_VERSION}&timeout=90s"
+          echo Deploying CodeFlare operator
+          IMG="${REGISTRY_ADDRESS}"/codeflare-operator
+          make image-push -e IMG="${IMG}"
+          make deploy -e IMG="${IMG}"
+          kubectl wait --timeout=120s --for=condition=Available=true deployment -n openshift-operators codeflare-operator-manager
+          echo Deploying MCAD controller
+          kubectl create ns codeflare-system
+          cat <<EOF | kubectl apply -n codeflare-system -f -
+          apiVersion: codeflare.codeflare.dev/v1alpha1
+          kind: MCAD
+          metadata:
+            name: mcad
+          spec:
+            controllerResources: {}
+          EOF
+          cat <<EOF | kubectl apply -n codeflare-system -f -
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: ClusterRole
+          metadata:
+            name: mcad-controller-rayclusters
+          rules:
+            - apiGroups:
+                - ray.io
+              resources:
+                - rayclusters
+                - rayclusters/finalizers
+                - rayclusters/status
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+          EOF
+          cat <<EOF | kubectl apply -n codeflare-system -f -
+          kind: ClusterRoleBinding
+          apiVersion: rbac.authorization.k8s.io/v1
+          metadata:
+            name: mcad-controller-rayclusters
+          subjects:
+            - kind: ServiceAccount
+              name: mcad-controller-mcad
+              namespace: codeflare-system
+          roleRef:
+            apiGroup: rbac.authorization.k8s.io
+            kind: ClusterRole
+            name: mcad-controller-rayclusters
+          EOF
+          kubectl wait --timeout=120s --for=condition=Available=true deployment -n codeflare-system mcad-controller-mcad
+
+          GOFLAGS="" make test-e2e
+      from: test-bin
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```

Comment:
Based on https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md I would consider 4 CPU and at least 4Gi memory (considering just MCAD, Ray and CodeFlare operator now).